hub/api/dataset.py

```diff
@@ -106,6 +108,9 @@ def init(
 
         Raises:
             AgreementError: When agreement is rejected
+            UserNotLoggedInException: When user is not logged in
+            InvalidTokenException: If the specified toke is invalid
```

Comment:
Typo, should be changed to `InvalidTokenException: If the specified token is invalid`